index.html

```diff
@@ -0,0 +1,688 @@
+            <img class="headline-img" src="images/newmoonrocket.jpg" alt="new moon rocket">
+        <figcaption><a href="#"><b>8 Minutes of Fire: NASA's 2nd Test of Giant New Moon Rocket Is a Success</b></a></figcaption>
+        </figure></div>
+        <div class="headbox">
+            <figure><img class="headline-img" src="images/lastrocket.jpg" alt="last rocket">
+            <figcaption><a href="#"><b>NASA's Last Rocket</b></a></figcaption>
+            </figure></div>
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/china-russia.jpg" alt="china russia">
+        <figcaption><a href="#"><b>China and Russia Agree To Explore The Moon Together</b></a></figcaption>
+        </figure></div>
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/spaceX.jpg" alt="spaceX rocket">
+        <figcaption><a href="#"><b>SpaceX Mars Rocket Prototype Explodes, But This Time It Lands First</b></a></figcaption>
+        </figure></div>
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/Husband.jpg" alt="Husband">
+        <figcaption><a href="#"><b>'When They Have Finished, My Husband Hands Them A 20'</b></a></figcaption>
+        </figure></div>
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/sharpshooter.jpg" alt="sharpshooter">
+        <figcaption><a href="#"><b>An 89-Year-Old Sharpshooter Takes Aim At India's Patriarchy</b></a></figcaption>
+        </figure></div>
+        <div class="headbox">
+            <figure>
+            <img class="headline-img" src="images/guru.jpg" alt="guru">
+        <figcaption><a href="#"><b>'Guru, Genius, Gentlemen, Scout</b></a></figcaption>
+        </figure></div>
+    </div>
+    
+    <footer>
+        <div class="footer-logo">
+            <a href="#"><img class="nyt-logo-2" src="images/592fe3133919fe0ee3614d93.png" alt="newyorktimes logo"></a>
+        </div>
+        <p class="homepage"><a href="#">Go to Homepage</a></p>
+        <div class="container">
+            <div class="dir box-1">
+                <ul>
+                    <li><a href="#">
+                            <h3>NEWS</h3>
+                        </a></li>
+                    <li><a href="#">Homepage</a></li>
+                    <li><a href="#">World</a></li>
+                    <li><a href="#">Coronavirus</a></li>
+                    <li><a href="#">U.S</a></li>
+                    <li><a href="#">Politics</a></li>
+                    <li><a href="#">New York</a></li>
+                    <li><a href="#">Business</a></li>
+                    <li><a href="#">Tech</a></li>
+                    <li><a href="#">Science</a></li>
+                    <li><a href="#">Sports</a></li>
+                </ul>
+            </div>
+            <div class="dir box-2">
+                <ul>
+                    <li><a href="#">
```

Comment:
- [ ] For your entire footer section, please have a look at the original and make appropriate changes

![footer](https://user-images.githubusercontent.com/86371332/124489912-6940d480-dda9-11eb-8fb1-2fa826c278a8.png)

### TIP: Don't center your text, you can try aligning your header or the container to center